.env.example

```
@@ -16,6 +16,22 @@ USERDIR=/home/username
+#   https://haugene.github.io/docker-transmission-openvpn/supported-providers/
+OPENVPN_PROVIDER=YOUR_PROVIDER
+OPENVPN_CONFIG=your_server_config
+OPENVPN_USERNAME=your_vpn_username
+OPENVPN_PASSWORD=your_vpn_password
+# CIDR(s) of networks that should bypass the VPN tunnel. Must include the
+# subnet your machine is on, or the web UI on :9091 will be unreachable.
+# Comma-separated for multiple networks (e.g. "192.168.0.0/16,10.0.0.0/8").
+LOCAL_NETWORK=192.168.0.0/16
+# Optional Transmission web UI auth. Leave blank for no auth.
+TRANSMISSION_USERNAME=
+TRANSMISSION_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRANSMISSION_PASSWORD key should go before the TRANSMISSION_USERNAME key

(UnorderedKey)